### PR TITLE
Feature/cleanup filter template

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
Templates cleanup.

**Comment by Mitso** [comment]
@alexmuller please review. This is a small cleanup of templates and updated test.

File: molo/commenting/tests/test_views.py [hunk 1]
```diff
@@ -610,13 +610,19 @@ def test_notification_reply_list(self):
...
+            ntfy.find("p").get_text().strip() in [
+                'You have 1 unread reply',
+                'You have 2 unread replies'
+            ])
 
         # Unread notifications
         response = self.client.get(
             reverse('molo.commenting:reply_list'))
-        self.assertContains(response, 'You have 1 unread replies')
+        self.assertTrue(response, [
```
Comment (codiebeulaine) [inline molo/commenting/tests/test_views.py:622]: @Mitso This should be response in [], instead of response, [] ??

**Comment by codiebeulaine** [comment]
@Mitso This all looks good, made a tiny comment though

**Comment by codiebeulaine** [comment]
Nice backend work @Mitso :)

**Pull request merged**